libindy_vdr/src/ffi/mod.rs

```diff
@@ -72,6 +77,25 @@ pub extern "C" fn indy_vdr_set_cache_directory(path: FfiStr) -> ErrorCode {
     }
 }
 
+#[no_mangle]
+pub extern "C" fn indy_vdr_set_ledger_txn_cache(
+    capacity: usize,
+    expire_offset: u64,
+    path_opt: FfiStr,
+) -> ErrorCode {
+    catch_err! {
+        debug!("Setting pool ledger transactions cache: capacity={}, expire_offset={}", capacity, expire_offset);
+        let store = match path_opt.as_opt_str() {
+            Some("") => OrderedHashMap::new(new_mem_ordered_store()),
```

Comment:
```suggestion
            None | Some("") => OrderedHashMap::new(new_mem_ordered_store()),
```
or you could match `path_opt.as_opt_str().unwrap_or_default()`